extensions/ql-vscode/src/view/data-extensions-editor/MethodRow.tsx

```diff
@@ -59,11 +61,12 @@ type Props = {
 };
 
 export const MethodRow = (props: Props) => {
-  const { externalApiUsage, modeledMethod } = props;
+  const { externalApiUsage, modeledMethod, modifiedSignatures } = props;
 
   const methodCanBeModeled =
     !externalApiUsage.supported ||
-    (modeledMethod && modeledMethod?.type !== "none");
+    (modeledMethod && modeledMethod?.type !== "none") ||
+    modifiedSignatures.has(externalApiUsage.signature);
```

Comment:
This change is attempting to mitigate a but where when you change a method from "modeled" to "unmodeled" and then save it'll temporarily show as "modeled by codeql or another pack". This bug appears on main as well and is in fact more noticeable there. I'm going to look into this and I'm hoping to address it separately.